### PR TITLE
SUT switch - https://github.com/ehrbase/project_management/issues/117

### PR DESCRIPTION
**Pull request by testautomation**
resolves https://github.com/ehrbase/project_management/issues/117

- added ~Test Environment~ SUT (system under test) configs to suite_settings
- target SUT can now be defined in suite_settings.robot and switching requires only to change `${SUT}` variable, which can be done in suite_settings or from command line
- added a command line option to allow manual control of the ~test environment~ SUT (`-v nodocker`)
- updated resource files:
  - composition_keywords
  - directory_keywords
  - ehr_keywords
  - generic_keywords
- fixed typo in a KW in DIRECTORY_TESTS
- updated/fixed my Taskfile.yml

### CODE REVIEW

**Comment by testautomation** [comment]
@testautomation  TODO
- [ ] Change SWAGGER to HEARTBEAT
- [ ] Use different URL for heartbeat / sanity check  (i.e.

**Comment by testautomation** [comment]
@ppazos  please review and test `-v nodocker` on your host

File: tests/README.md [hunk 1]
```diff
@@ -1,60 +1,76 @@
...
-> PREREQUISITES
->
-> - Docker, Python 3 with Pip are installed
-> - RF dependencies are installed (`cd tests/`, `pip install -r requirements.txt`)
-> - **No DB / no server running!**
-> - ports `8080` and `5432` not used by any other application! (`netstat -tulpn`)
+---
 
+## Prerequisites
+xcvzxcvz
```
[the remark this answers is not in the document]
Comment (testautomation) [reply tests/README.md]: no, it's a typo 😃

**Comment by ppazos** [comment]
@testautomation looks OK :+1:

**Pull request merged**